packages/grafana-ui/src/components/Table/BarGaugeCell.tsx

```diff
@@ -37,8 +38,20 @@ export const BarGaugeCell: FC<TableCellProps> = (props) => {
     barGaugeMode = BarGaugeDisplayMode.Basic;
   }
 
-  return (
-    <div {...cellProps} className={tableStyles.cellContainer}>
+  const getLinks = () => {
+    if (field.getLinks === undefined) {
```

Comment:
```suggestion
    if (!isFunction(field.getLinks)) {
```

Reply:
Thank you for the tip! Learned something new about lodash today! :D